core/js/src/main/scala/zio/internal/PlatformLive.scala

```diff
@@ -40,7 +40,7 @@ object PlatformLive {
       }
 
       def reportFailure(cause: Cause[Any]): Unit =
-        if (!cause.interruptedOnly)
+        if (!cause.interruptedOnly && cause.died)
```

Comment:
Isn't `!cause.interruptedOnly` redundant now?

Reply:
Yes, you are right! Will fix that in a moment.

Reply:
🤔 `!cause.interruptedOnly` is true if either the fiber died or failed, so `!cause.interruptedOnly && cause.died` would fail if the fiber failed.

Reply:
So yeah 😅